support/mender-client.service

```diff
@@ -8,7 +8,7 @@ Conflicts=mender.service
 Type=idle
 User=root
 Group=root
-ExecStart=/usr/bin/mender daemon
+ExecStart=/usr/bin/mender --no-syslog daemon
```

Comment:
We should not do this here, only in `meta-mender` for the branches that this behavior changed (`master` and `kirkstone`).

@kacf and @olepor have the last word :+1:

Reply:
@lluiscampos: We agreed yesterday to do it here because since this is the systemd service file, it is guaranteed that the logs are always available using journald. It's possible that someone who were previously using a separate syslog daemon won't see the logs there anymore, which is a behavior change, but we concluded that it is justified since nothing will be lost, and logs are just an informational tool. We do want to keep it in master only though, so at least you won't get this surprise in a patch release.

Reply:
Thanks for the explanation